tee.md

```diff
@@ -0,0 +1,54 @@
+# Deploy on Phala Cloud
```

Comment:
looks like there are some linting issues here. Can we run `bun run format:fix`

Reply:
Fixed